### ISSUE TITLE
Future of this package

### ISSUE THREAD

**Issue by jbusecke** [jbusecke/cmip6_omz#60]
Hey @sditkovsky. Was this package actually used in our recent paper? 
I am trying to trim down the amount of packages to maintain, and I think this one has been stale for a bit now. 

If you do not object, I would archive it (which would not get rid of it, but deactivate new issue, CI etc).

**Comment by sditkovsky** [comment]
Yes, feel free to archive. I've been using the omz_boundaries() function, including in our recent paper, for calculating OMZ volume. But no reason to keep it active for maintenance.

**Comment by jbusecke** [comment]
Hmm we could also consider refactoring the useful partsto somewhere else?
